account/identity.go

```diff
@@ -138,6 +142,28 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
+// Exists returns true|false where an object exists with an identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
+
+	queryStmt, err := m.db.CommonDB().Prepare(fmt.Sprintf(`
```

Comment:
What's the reasoning behind the shared statement? You could easily query for existence using:

```go
	query := fmt.Sprintf("SELECT EXISTS(SELECT id FROM %s WHERE id=$1 AND deleted_at IS NULL)", m.TableName())
	err := m.db.CommonDB().QueryRow(query, id.String()).Scan(&exists)
```